addons/mass_mailing/tests/__init__.py

```diff
@@ -6,3 +6,4 @@
 from . import test_mailing_list
 from . import test_mailing_controllers
 from . import test_mailing_mailing_schedule_date
+from . import test_mailing_saving
```

Comment:
Would name it ``test_mailing_ui`` as it is generally the convention when having some testing tours.